libshaderc/src/shaderc.cc

```diff
@@ -23,6 +23,7 @@
 
 #include "libshaderc_util/compiler.h"
 #include "libshaderc_util/resources.h"
+#include "SPIRV/spirv.hpp"
```

Comment:
Our style requires this be moved up, as "other libraries' .h file":

http://google-styleguide.googlecode.com/svn/trunk/cppguide.html#Names_and_Order_of_Includes